leader-election-api/src/main/java/com/palantir/leader/LeaderElectionService.java

```diff
@@ -81,4 +82,6 @@
      * @return true if and only if this node was able to cause itself to lose leadership
      */
     boolean stepDown();
+
+    Optional<URL> getCurrentLeader();
```

Comment:
This is too strongly named - I'd prefer `getCurrentSuspectedLeader`, as this may not give you the true identity of the leader (e.g. consider what happens if a leadership proposal is not learned on all learners). Probably also want javadoc explaining what this doesn't guarantee.